integration-tests/utils/zendesk/showZendeskTicket.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import { authoriseAs } from './authoriseAs'
+import {
+  ZENDESK_AGENT_EMAIL,
+  ZENDESK_BASE_URL,
+  ZENDESK_TICKETS_ENDPOINT
+} from '../../constants/zendeskParameters'
+
+export const showZendeskTicket = async (ticketId: string) => {
```

Comment:
Is there a reason you're not just calling this `getZendeskTicket`? Calling it "show" makes me think there's some UI or something. Pedantic point though...

Reply:
That's what they call it the API docs! But I see your point, I prefer `getZendeskTicket` so will rename!